sympy/matrices/tests/test_matrices.py

```diff
@@ -17,6 +17,9 @@
 from sympy.utilities.pytest import raises, XFAIL, slow, skip
 from sympy.solvers import solve
 from sympy.assumptions import Q
+import warnings
+from sympy.utilities.exceptions import SymPyDeprecationWarning
+warnings.filterwarnings("ignore", category=SymPyDeprecationWarning)
```

Comment:
@siefkenj I didn't change that because @asmeurer suggested,
>there can be one test to make sure the warning itself works.

in Gitter. Anyways I'll remove that.